knative-operator/pkg/controller/knativeserving/servicemesh/servicemesh.go

```diff
@@ -239,6 +242,39 @@ func isServiceMeshMemberRollReady(servingNamespace string, api client.Client) (b
 	return false, nil
 }
 
+// create wide-open networkpolicies for the service mesh components
+func installNetworkPolicies(instanceNamespace string, api client.Client) error {
+	log.Info("Installing Mesh Network Policies")
+	namespace := ingressNamespace(instanceNamespace)
+	const path = "deploy/resources/serviceMesh/networkpolicies.yaml"
+
+	manifest, err := mf.NewManifest(path, false, api)
+	if err != nil {
+		log.Error(err, "Unable to create Mesh Network Policy install manifest")
+		return err
+	}
+	smcp := &maistrav1.ServiceMeshControlPlane{}
+	ready, err := isServiceMeshControlPlaneReady(instanceNamespace, api)
```

Comment:
Do we really need to wait for it becoming ready? I feel like we could just add the networkpolicy once the SMCP is there.

Reply:
I agree. If we can `Get` the resource, we should install the NP, regardless of the former's status.

Reply:
Added **isServiceMeshControlPlaneReady** in order to make sure that before creating networkpolicy SMCP is ready

But i can just do GET operation on SMCP in order to get the data for `InjectOwner`